autouri/gcsuri.py

```diff
@@ -30,16 +31,65 @@
 logger = logging.getLogger(__name__)
 
 
+ENV_VAR_GOOGLE_APPLICATION_CREDENTIALS = "GOOGLE_APPLICATION_CREDENTIALS"
+
+
+def google_auth(service_account_key_file):
```

Comment:
Maybe we could give this function a more descriptive name like `add_google_authentication_credentials_to_environment`?

Reply:
Thanks, renamed it to `add_google_app_creds_to_env()`.

Reply:
Great!